commands/sync.md

```diff
@@ -1,3 +1,14 @@
-@examples
+Initiates a replication stream from the master.
+
+The `SYNC` command is called by Redis replicas for initiating a replication
+stream from the master. It has been replaced in newer versions of Redis by
+ `PSYC`.
```

Comment:
PSYC->PSYNC?

Reply:
Dang this fangled butterfly fruity keyboard - thanks!